server/sonar-web/src/main/js/apps/projects/styles.css

```diff
@@ -136,10 +137,18 @@
   content: "";
 }
 
+.project-card-measure.pull-right {
```

Comment:
In fact I didn't use .`text-right` because it's overwritten by the `text-align: center` of `.project-card-measure`.
So either I let it like right now, or I add a `!important` to `.text-right`, but I'm sure it's a good idea.